docs/framework-wrappers.md

```diff
@@ -10,7 +10,7 @@ path: /docs/framework-wrappers/
 
 Material Components for the web are architected to be adaptable to various major web frameworks. The following wrapper libraries are available:
 
-  - [Material Components for React](https://github.com/material-components/material-components-web-react): MDC Web integration for React (using [foundations/adapters](./integrating-into-frameworks.md#the-advanced-approach-using-foundations-and-adapters))
+  - [Material Components for React](https://github.com/material-components/material-components-web-react): MDC Web integration for React (using [foundations/adapters](./integrating-into-frameworks.md#the-advanced-approach-using-foundations-and-adapters)). *Please note that this project is no longer under active development.*
```

Comment:
Thanks for sending the PR!

Can you please move this bullet point below? I feel the active ones should stay on top.

Reply:
That makes sense. Done with the change.